program-test/src/lib.rs

```diff
@@ -698,3 +729,59 @@ impl ProgramTestBanksClientExt for BanksClient {
         ))
     }
 }
+
+struct DroppableTask<T>(Sender<()>, JoinHandle<T>);
+
+impl<T> Drop for DroppableTask<T> {
+    fn drop(&mut self) {
+        if let Err(e) = self.0.try_send(()) {
+            debug!("Error cancelling task: {:?}", e);
+        }
+    }
+}
+
+pub struct ProgramTestContext {
+    pub banks_client: BanksClient,
+    pub payer: Keypair,
+    pub last_blockhash: Hash,
+    _bank_task: DroppableTask<()>,
+}
+
+impl ProgramTestContext {
+    fn new(
+        bank_forks: Arc<RwLock<BankForks>>,
+        banks_client: BanksClient,
+        payer: Keypair,
+        last_blockhash: Hash,
+        genesis_config: GenesisConfig,
+    ) -> Self {
+        // Run a simulated PohService to provide the client with new blockhashes.  New blockhashes
+        // are required when sending multiple otherwise identical transactions in series from a
+        // test
+        let target_tick_duration = genesis_config.poh_config.target_tick_duration;
+        let (tx, mut rx) = mpsc::channel(1);
```

Comment:
```suggestion
        let (sender, mut receiver) = mpsc::channel(1);
```
🙏🏼.   Most people unfortunately like to use tx to mean transaction in the code base, except when it means something else like in this case

Reply:
Ah, right, on it